swf/src/types.rs

```diff
@@ -417,7 +417,7 @@ pub enum BlendMode {
 /// [SWF19 pp.37-38 ClipActionRecord](https://www.adobe.com/content/dam/acom/en/devnet/pdf/swf-file-format-spec.pdf#page=37)
 #[derive(Debug, Clone, PartialEq)]
 pub struct ClipAction {
-    pub events: EnumSet<ClipEventFlag>,
+    pub events: BigEnumSet<ClipEventFlag>,
```

Comment:
Feels to me like this just takes extra memory for no benefit. (previously it was stored in 4 bytes just like in the SWF itself, now it's stored in `usize`)

Reply:
Yes sorry, I modified this lib to compile with rust 1.50 in another project

Reply:
enumset is not compatible